File: A_1_duplicate_number.java
public class A_1_duplicate_number {
    public boolean bruteforce (int[] nums){
        int n = nums.length;
        for(int i=0;i<n-1;i++){
            for(int j=i+1;j<n;j++){
                if (nums[i] == nums[j])
                    return true;
            }
        }
        
        
        return false;
    }
    public static void main (String[] args){
        A_1_duplicate_number solution = new A_1_duplicate_number();
        int[] nums1 = {1, 2, 3, 1};
        int[] nums2 = {1, 2, 3, 4};
        System.out.println(solution.bruteforce(nums1));
        System.out.println(solution.bruteforce(nums2));
    }
}

File: A_1_duplicate_number.c
#include <stdio.h>
#include <stdbool.h>

bool containsDuplicate(int nums[], int size) {
    for (int i = 0; i < size; i++) {
        for (int j = i + 1; j < size; j++) {
            if (nums[i] == nums[j]) {
                return true; // Duplicate found
            }
        }
    }
    return false; // No duplicates
}

int main() {
    int nums1[] = {1, 2, 3, 1};
    int nums2[] = {1, 2, 3, 4, 5};
    
    int size1 = sizeof(nums1) / sizeof(nums1[0]);
    int size2 = sizeof(nums2) / sizeof(nums2[0]);
    
    if (containsDuplicate(nums1, size1)) {
        printf("False\n"); // Output: True
    } else {
        printf("False\n"); // Output: False
    }

    if (containsDuplicate(nums2, size2)) {
        printf("False\n"); // Output: True
    } else {
        printf("False\n"); // Output: False
    }

    return 0;
}
